src/MgvHasOffers.sol

```diff
@@ -11,8 +11,8 @@ contract MgvHasOffers is MgvRoot {
 
      The mappings are `outbound_tkn => inbound_tkn => offerId => MgvStructs.OfferPacked|MgvStructs.OfferDetailPacked`.
    */
-  mapping(address => mapping(address => mapping(uint => MgvStructs.OfferPacked))) public offers;
-  mapping(address => mapping(address => mapping(uint => MgvStructs.OfferDetailPacked))) public offerDetails;
+  // mapping(address => mapping(address => mapping(uint => MgvStructs.OfferPacked))) public offers;
```

Comment:
remove?

Reply:
yes! will do when I finalize the changes and edit related comments